src/test/resources/fixtures/additional-fixtures/template-welsh/output.txt

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
This is a different case to other tests in `additional-fixtures` because the HTML is *supposed* to be different to `govuk-frontend`.  AFAIK all existing tests in there specify HTML which is supposed to match the output from `govuk-frontend` - I'd like to discuss how to differenciate these cases as I'd like to remove the `output.txt` from the other cases and have it rendered by the template renderer instead.

Reply:
I investigated and it was really easy to move the patched fixture to a new folder so I've created a `patched-fixtures` folder and added a README to the fixtures folder linking back to maintainers section on unit tests and just describing each of the folders with a bit more context. Felt like it was good to add this next to the folders so it was easier to get a quick reminder if making changes.